views/messages/index.erb

```diff
@@ -0,0 +1,5 @@
+<ul>
+  <% @messages.each do |message| %>
+    <li><%= message %></li>
+  <% end %>
+</ul>
```

Comment:
It's not in the requirements but it might be nice to include a link to the new message form ;)